[Edited] Remove term in Purchase Receipt and edit line in Sales Billing

Saralrat opened the pull request.

kittiu commented:
@Saralrat what is changed?
Note: for Frappe, it is not easy to see from the diff. So, picture will be good here.

Saralrat commented:
> @Saralrat what is changed? Note: for Frappe, it is not easy to see from the diff. So, picture will be good here.

I have removed the section showing the term in Purchase Receipt form.

![Screenshot from 2024-10-10 14-59-31](https://github.com/user-attachments/assets/6736fca5-0a47-44b6-a3a8-d0df8b145218)
 
 and Increased the spacing between notes and footers and fixed the company name link in Sales Billing form.
 
![Screenshot from 2024-10-10 15-02-42](https://github.com/user-attachments/assets/3902cf9c-9cb1-45e7-8fce-18b0e3940c3d)
